README.md

```diff
@@ -42,22 +42,57 @@ These tasks are available via **Terminal ▸ Run Task...** and **Terminal ▸ Ru
+The extension adds the following commands, available via the command palette.
+
+#### Configuration
+
+- **Create New Project...**: Create a new Swift project using a template. This opens a dialog to guide you through creating a new project structure.
+- **Create New Swift File...**: Create a new `.swift` file in the current workspace.
+- **Select Toolchain**: Select which locally installed Swift toolchain (including Xcode toolchains on macOS) you want to use Swift tools from.
+
+The following command is only available on macOS
+
+- **Select Target Platform**: This is an experimental command used to get code completion for iOS, tvOS projects
+
+#### Building and Debugging
+
+- **Run Build**: Run `swift build` for the package associated with the open file.
+- **Debug Build**: Run `swift build` with debugging enabled for the package associated with the open file, launching the binary and attaching the debugger.
+- **Attach to Process...**: Attach the debugger to an already running process for debugging.
+- **Clean Build Folder**: Clean the build folder for the package associated with the open file, removing all previously built products.
+
+### Diagnostics
+
+- **Capture VS Code Swift Diagnostic Bundle**: Capture a diagnostic bundle from VS Code, containing logs and information to aid in troubleshooting Swift-related issues.
+- **Clear Diagnostics Collection**: Clear all collected diagnostics in the current workspace to start fresh.
+- **Restart LSP Server**: Restart the Swift Language Server Protocol (LSP) server for the current workspace.
+- **Re-Index Project**: Force a re-index of the project to refresh code completion and symbol navigation support.
+
+#### Dependency Management
 
 - **Resolve Package Dependencies**: Run `swift package resolve` on package associated with open file.
 - **Update Package Dependencies**: Run `swift package update` on package associated with open file.
 - **Reset Package Dependencies**: Run `swift package reset` on package associated with open file.
-- **Open Package.swift**: Open Package.swift for package associated with open file.
+- **Add to Workspace**: Add the current package to the active workspace in VS Code.
 - **Clean Build**: Run `swift package clean` on package associated with open file.
+- **Open Package.swift**: Open Package.swift for package associated with open file.
+- **Use Local Version**: Switch the package dependency to use a local version of the package instead of the remote repository version.
+- **Edit Locally**: Make the package dependency editable locally, allowing changes to the dependency to be reflected immediately.
+- **Revert To Original Version**: Revert the package dependency to its original, unedited state after local changes have been made.
+- **View Repository**: Open the external repository of the selected Swift package in a browser.
+
+### Testing
+
+- **Run Multiple Times...**: Run the current set of tests multiple times and display the results.
```

Comment:
are these available from the command palette? I'd argue we only should put palette commands here, not context ones

Reply:
Done